src/components/pages/PendingApplications/ApplicationModal.js

```diff
@@ -34,11 +33,34 @@ const ApplicationModal = ({
   const handleCancel = () => {
     setDisplayModal(false);
     setProfileId('');
-    setHideForm(true);
   };
 
-  const displayForm = () => {
-    setHideForm(false);
+  const openNotificationWithIcon = (type, status, err) => {
```

Comment:
Good work ensuring that users get feedback for actions taken on the app - we'll want this kind of interactivity moving forward to ensure our app feels modern and is a smooth experience to use.